Deploy-CAM.ps1

```diff
@@ -2602,7 +2602,7 @@ $subscriptionsToDisplay = $subscriptions | Where-Object { $_.State -eq 'Enabled'
 
 $chosenSubscriptionIndex = $null
 if ($subscriptionsToDisplay.Length -lt 1) {
-    Write-Host ("Account " + $rmContext.Account.Id + " has access to no enabled subscriptions. Exiting.")
+    Write-Host ("Account " + $rmContext.Account.Id + " has access to no enabled subscriptions. Exiting.") -ForegroundColor Red -BackgroundColor Yellow
```

Comment:
I think this particular combination of `Write-Host` colors could be extracted as a method, which would make the intent clear as well as de-clutter the rest of the script a bit.

Reply:
Made a function

Reply:
Yeah looks better with the function thanks!